ext/tidy/tidy.c

```diff
@@ -291,10 +291,12 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_parse_string, 0, 0, 1)
 	ZEND_ARG_INFO(0, encoding)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_error_buffer, 0)
+ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_get_error_buffer, 0, 0, 1)
+    ZEND_ARG_INFO(0, object)
```

Comment:
Should I had used [`ZEND_ARG_OBJ_INFO`](https://github.com/php/php-src/blob/master/Zend/zend_API.h#L101)? :sweat_smile: